.github/workflows/build.yml

```
@@ -0,0 +1,63 @@
+name: Build and Test
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main, beta ]
+  pull_request:
+    branches: [ main, beta ]
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+env:
+  NODE_VERSION: 18
+  CACHE_KEY: '${{ github.ref }}-${{ github.run_id }}-${{ github.run_attempt }}'
+
+jobs:
+
+  build:
+    strategy:
+      matrix:
+        node: [ 16, 18, 20 ]
+
+    name: Build Package (Node ${{ matrix.node }})
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci --include=dev
```

What does this `--include` flag do? Will this only install development dependencies?

---

https://stackoverflow.com/a/64648870/1370149

---

Ahh, but then it can be omitted here no? Since `NODE_ENV` is never set.
```suggestion
        run: npm ci
```